storybook/stories/settings/general/General.stories.js

```diff
@@ -22,6 +22,8 @@ import SupportSettings from '../../../../source/renderer/app/components/settings
 import TermsOfUseSettings from '../../../../source/renderer/app/components/settings/categories/TermsOfUseSettings';
 import WalletsSettings from '../../../../source/renderer/app/components/settings/categories/WalletsSettings';
 
+// Data
```

Comment:
@aleksandardjordjeviciohk please remove this.

Reply:
Updated